.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous integration
+on: push
```

Comment:
(Sorry, I should have noticed this before pushing merge button...) if we'd like to trigger a job on creating pull request, writing like `on: [push, pull_request]` here might be better?

Reply:
I think it could be enough with only `push` but I will add `pull_request` if the action is not triggered by a next new PR. Thanks.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows

Reply:
Apparently, Naoya was right. My latest pull request (#131) did not trigger CI action in this repository. It was only triggered in my fork.

Reply:
I have updated the configuration in #133. Thanks!